packages/cli/src/commands/auth/login/login.command.ts

```diff
@@ -0,0 +1,69 @@
+import fastify from 'fastify';
+import fastifyStatic from '@fastify/static';
+import path from 'path';
+import fs from 'fs';
+import open from 'open';
+import crypto from 'crypto';
```

Comment:
need to be "node:crypto" for all node default modules

Reply:
Done